locales/pt_BR.json

```diff
@@ -0,0 +1,777 @@
+		},
+		"avoid-inline-spacing": {
+			"description": "Certifique-se de que o espaçamento de texto definido via atributos de estilo pode ser ajustado com folhas de estilo personalizadas",
+			"help": "O espaçamento de texto 'inline' deve ser ajustável com folhas de estilo personalizadas"
+		},
+		"blink": {
+			"description": "Certifique-se de que elementos <blink> não são utilizados",
+			"help": "Elementos <blink> foram descontinuados e não devem ser utilizados"
+		},
+		"button-name": {
+			"description": "Certifique-se de que botões tenham texto discernível",
+			"help": "Botões devem ter texto discernível"
+		},
+		"bypass": {
+			"description": "Certifique-se de que cada página tenha ao menos um mecanismo para o usuário pular a navegação e ir direto ao conteúdo",
+			"help": "A página deve ter meios para pular blocos repetidos"
+		},
+		"checkboxgroup": {
+			"description": "Certifique-se de que elementos <input type=\"checkbox\"> relacionados tenham um grupo e que a designação do grupo seja consistente",
+			"help": "Caixas de seleção com o mesmo valor do atributo 'name' devem ser parte de um grupo"
+		},
+		"color-contrast": {
+			"description": "Certifique-se de que o contraste entre as cores de primeiro plano e de fundo atenda aos limites de relação de contraste WCAG 2 AA",
+			"help": "Os elementos devem ter contraste de cor suficiente"
+		},
+		"css-orientation-lock": {
+			"description": "Certifique-se de que o conteúdo não está bloqueado para nenhuma orientação de tela específica, e que o mesmo seja operável em todas as orientações de tela",
+			"help": "CSS Media queries não são usadas para bloquear a orientação de tela"
+		},
+		"definition-list": {
+			"description": "Certifique-se de que elementos <dl> estão estruturados corretamente",
+			"help": "Elementos <dl> devem conter diretamente apenas grupos <dt> e <dd> propriamente ordenados, ou elementos <script> ou <template>"
+		},
+		"dlitem": {
+			"description": "Certifique-se de que elementos <dt> e <dd> estão contidos em um elemento <dl>",
+			"help": "Elementos <dt> e <dd> devem estar contidos em um elemento <dl>"
+		},
+		"document-title": {
+			"description": "Certifique-se de que cada documento HTML contenha um elemento <title> não vazio",
+			"help": "Documentos devem ter um elemento <title> para ajudar na navegação"
+		},
+		"duplicate-id-active": {
+			"description": "Certifique-se de que cada valor do atributo 'id' de elementos ativos seja único",
+			"help": "IDs de elementos ativos devem ser únicos"
+		},
+		"duplicate-id-aria": {
+			"description": "Certifique-se de que cada valor do atributo 'id' usado no ARIA e em 'labels' seja único",
+			"help": "IDs usados no ARIA e em 'labels' devem ser únicos"
+		},
+		"duplicate-id": {
+			"description": "Certifique-se de que cada valor do atributo 'id' seja único",
+			"help": "O valor do atributo 'id' deve ser único"
+		},
+		"empty-heading": {
+			"description": "Certifique-se de que títulos tenham texto discernível",
```

Comment:
This might generate some confusion in pt_BR. Both words, header and heading, can be translated as 'cabeçalho'. So, many programs, like MS Word and all text editors I can remember, use 'título' to refer to headings, leaving 'cabeçalho' to headers.
So, in this case, it would be better to retain 'título'.